Dockerfile-dev

```diff
@@ -5,7 +5,7 @@ LABEL io.k8s.display-name="OpenShift Origin Builder" \
 
 RUN INSTALL_PKGS=" \
       which tar wget hostname sysvinit-tools util-linux \
-      socat tree findutils lsof bind-utils \
+      socat tree findutils lsof bind-utils fuse-overlayfs\
```

Comment:
nit - can we reorganize these to be in alpha order (looks like there are dupes)

Reply:
Done